cumulus/tasks/parse-pdr/tests/parse_pdrs_test.js

```diff
@@ -5,6 +5,7 @@ const test = require('ava');
 const testUtils = require('@cumulus/common/test-utils');
 const errors = require('@cumulus/common/errors');
 const modis = require('@cumulus/test-data/payloads/new-message-schema/parse.json');
+const { cloneDeep } = require('lodash');
```

Comment:
This is missing from `package.json`. I suggest we use `lodash.cloneDeep` instead of the entire lodash library